ansible/roles/beegfs/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Ensure the BeeGFS package repo is defined
+  get_url:
+    url: "{{ beegfs_repo_url }}"
+    dest: /etc/yum.repos.d/beegfs-rhel7.repo
+  become: True
+
+- name: Ensure any required kernel modules are loaded
+  modprobe:
+    name: "{{ item }}"
+    state: present
+  become: True
+  with_items: "{{ beegfs_kmod_preload }}"
```

Comment:
need a default for this?

Reply:
This won't be persistent across reboots. Here's an example of making them persistent: https://github.com/michaelrigart/ansible-role-interfaces/blob/master/handlers/main.yml